pkgs/by-name/gh/ghostty/package.nix

```
@@ -0,0 +1,139 @@
+{
+  lib,
+  stdenv,
+  bzip2,
+  callPackage,
+  expat,
+  fetchFromGitHub,
+  fontconfig,
+  freetype,
+  git,
+  glib,
+  gsettings-desktop-schemas,
+  gtk4,
+  harfbuzz,
+  libadwaita,
+  libGL,
+  libpng,
+  libX11,
+  libXcursor,
+  libXi,
+  libXrandr,
+  ncurses,
+  nix-update-script,
+  oniguruma,
+  pandoc,
+  pkg-config,
+  wrapGAppsHook4,
+  zig_0_13,
+  zlib,
+}:
+let
+  # Ghostty needs to be built with --release=fast, --release=debug and
+  # --release=safe enable too many runtime safety checks.
+  zig_hook = zig_0_13.hook.overrideAttrs {
+    zig_default_flags = "-Dcpu=baseline -Doptimize=ReleaseFast --color off";
+  };
+in
+stdenv.mkDerivation (final: {
+  pname = "ghostty";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ghostty-org";
+    repo = "ghostty";
+    tag = "v${final.version}";
+    hash = "sha256-AHI1Z4mfgXkNwQA8xYq4tS0/BARbHL7gQUT41vCxQTM=";
+  };
+
+  nativeBuildInputs = [
+    git
```

```suggestion
    git
    glib # Required for `glib-compile-schemas`
```
https://github.com/ghostty-org/ghostty/blob/35b9ceee2116331b83c0c86269394e2545070b0f/build.zig#L1387-L1394

---

Drew my eye: git is also no longer needed.